docs/azure_arc_jumpstart/azure_arc_k8s/day2/kind/local_kind_gitops_helm/_index.md

```diff
@@ -91,6 +91,8 @@ The demo application that will be deployed later in this guide relies on an ingr
 
     ![Running ingress nginx controller](./03.png)
 
+> **Note: If the ingress controller fails to start, you should redeploy the kind cluster with [kind_cluster_ingress.yaml](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/kind/kind_cluster_ingress.yaml) instead of kind_cluster.yaml.**
+
```

Comment:
```suggestion
> **Note: If the ingress controller fails to start, you should redeploy the kind cluster with [_kind_cluster_ingress.yaml_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/kind/kind_cluster_ingress.yaml) instead of _kind_cluster.yaml_.**

```